edgegrid/auth.go

```diff
@@ -23,6 +26,9 @@ type AuthParams struct {
 	headersToSign []string
 }
 
+// NewAuthParams is a convenience function for generating
+// an AuthParams{} from an *http.Request, acccess token string, client
+// token string, and a client secret string
 func NewAuthParams(req *http.Request, accessToken, clientToken, clientSecret string) AuthParams {
```

Comment:
@ChrisHines Thanks -- that is good feedback. I've made another commit, per your recommendation.